5gms/5G_APIs-overrides/M3_ContentHostingProvisioning.yaml

```diff
@@ -193,6 +193,13 @@ paths:
         '414':
           # URI too long
           $ref: 'TS29571_CommonData.yaml#/components/responses/414'
+        '422':
+          # Unprocessable Entity (e.g. syntactically invalid regular expression in request body)
+          description: Unprocessable Entity
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'TS29571_CommonData.yaml#/components/schemas/ProblemDetails'
```

Comment:
The 422 code may need to be added to Table 5.2.7.1-1 in TS 29.500 as it's not listed there as a valid SBI status code.